### ISSUE TITLE
Replace function: Doesn't always update existing fields?

### ISSUE THREAD

**Issue by gwtwod** [humio/issues#97]
Humio Version: 1.6.3--build-7700--sha-89765c301 (and 1.6.2)
Browser Version: Chrome Version 76.0.3809.100 (Official Build) (64-bit)

The `replace()` function seems a bit weird now, it doesnt always seem to modify existing fields?

Example search from the Sandbox tutorial dataset:

```
wikipedia products office
| replace(regex="office", with="playground", field=url, as=new_url)
| replace(regex="playground", with="bakery", field=new_url, as=new_url)
```

After this query I get these fields in the left-side field menu
```
"url": /products/office/92
"new_url" "/products/playground/92"
```

If I add a `| table([url, new_url])` at the end then I get the expected result:
```
"url": "/products/office/92"
"new_url": "/products/bakery/92"
```

I've also seen the same happen after trying to `replace()` values in a field I use with `timechart(field)`

**Comment by andejens** [comment]
Thx - we'll take a look

**Comment by gwtwod** [comment]
This still happens in version 1.6.5--build-7284--sha-9815b655e.

Heres two screenshots demonstrating the issue as well. I suspect the replace() function just doesn't update the field stats in the left-menu, so it looks like they haven't been changed. But if I try to use those fields in a search I can see they actually have been changed as expected.

![replace_bug_1](https://user-images.githubusercontent.com/40236320/67362427-46d23d80-f56b-11e9-83de-fc52d59d2c4d.png)

![replace_bug_2](https://user-images.githubusercontent.com/40236320/67362432-4b96f180-f56b-11e9-8cf5-9974934c6d41.png)

**Comment by andejens** [comment]
I apologise - no progress has been made - I have raised the priority internally.

**Comment by chvitved** [comment]
Thanks for reporting. This is fixed in 1.6.6
